src/v1/destinations/iterable/types.ts

```diff
@@ -0,0 +1,57 @@
+type FailedUpdates = {
+  invalidEmails?: string[];
+  invalidUserIds?: string[];
+  notFoundEmails?: string[];
+  notFoundUserIds?: string[];
+  invalidDataEmails?: string[];
+  invalidDataUserIds?: string[];
+  conflictEmails?: string[];
+  conflictUserIds?: string[];
+  forgottenEmails?: string[];
+  forgottenUserIds?: string[];
+};
+
+export type GeneralApiResponse = {
+  msg?: string;
+  code?: string;
+  params?: Record<string, unknown>;
+  successCount?: number;
+  failCount?: number;
+  invalidEmails?: string[];
+  invalidUserIds?: string[];
+  filteredOutFields?: string[];
+  createdFields?: string[];
+  disallowedEventNames?: string[];
+  failedUpdates?: FailedUpdates;
+};
+
+export type DestinationResponse = {
+  status: number;
+  response: GeneralApiResponse;
+};
+
+export type ResponseParams = {
```

Comment:
Lets make sure we have a common type lets not declare another type in networkHandler.ts also make sure we are representing the correct type we are ingesting here
https://github.com/rudderlabs/rudder-transformer/blob/9ce457588bb7e1c766d585193f8f15f49a2add1d/src/services/destination/nativeIntegration.ts#L203

Reply:
Also lets not use generic name Like ResponseParams but use integration specific name like `IterableResponseParams`

Reply:
done